prboom2/src/m_menu.c

```diff
@@ -1400,6 +1400,8 @@ menu_t MouseDef =
 
 #define MOUSE_SENS_MAX 100
 
+#define MOUSE_SENS_SCALE 8u
```

Comment:
This introduces a bug below where the signed/unsigned subtraction results in a large unsigned value when values < 0 are selected. Not sure why this needs to be unsigned - should it instead be signed?

Reply:
Ah whoops, I didn't want to do a wasteful signed division for the slider, but I unwisely made the constant unsigned and forgot.